src/TwigComponent/src/ComponentRendererInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\UX\TwigComponent;
+
+interface ComponentRendererInterface
+{
+    public function createAndRender(string $name, array $props = []): string;
+
+    public function render(MountedComponent $mounted): string;
```

Comment:
`MountedComponent` is marked as internal. Perhaps this method shouldn't be part of the interface? @weaverryan, WDYT?

Reply:
It *would* be easy to add later if we need it, and the use-case right now is *just* for needing `createAndRender`. So I'm +1 for that.

I'd also love a little bit of PHPdoc above the method, since it's now a public interface.